cli_reference/get_started_cli.adoc

```diff
@@ -105,13 +133,23 @@ There are a few ways to provide `osc` a configuration file. Basically when loadi
 .  The `.kubeconfig` file in the current directory
 .  The `.kubeconfig` file inside the `.kube` directory in the user's home: `~/.kube/.kubeconfig`
 
-For details about advanced management of config files, see link:../using_openshift/managing_cli_profiles.html[Managing CLI Profiles].
+You can link:setup_multiple_cli_profiles.html[configure and manage multiple CLI
+profiles] using the `osc config` command.
 
 == Projects
-A link:../architecture/openshift_model.html#project-experimental[project] in OpenShift terms is the package that can contain multiple link:../architecture/openshift_model.html[components] that will compose your logical "link:../v2_changes/applications.html[application]". 
+A link:../dev_guide/projects.html[project] in OpenShift is the package that can
+contain multiple
+link:../architecture/core_objects/openshift_model.html[components] that compose
+your logical "link:../whats_new/applications.html[application]".
 
-That said, most `osc` commands that will be described in the sections below run in the context of a _project_. When running `osc login` a project will be set as default to be used by every subsequent command, and if the user has access to more than one project, one of them will be picked:
+Most `osc` commands run in the context of a
+link:../dev_guide/projects.html[project]. When running `osc login`, a project is
+set as default to be used by every subsequent command, and if you have access to
+more than one project, one of them is picked:
 
```

Comment:
[Suggestion]
When you run the `osc login` command to configure the CLI, a default project is set to be used with subsequent commands. If you have access to multiple projects then one is selected, as shown in the following example.

[Example]
screen output